src/gui/Clipboard.cpp

```diff
@@ -46,15 +48,20 @@ void Clipboard::setText(const QString& text)
 {
     QClipboard* clipboard = QApplication::clipboard();
 
-#ifdef Q_OS_MAC
     QMimeData* mime = new QMimeData;
+#ifdef Q_OS_MAC
     mime->setText(text);
     mime->setData("application/x-nspasteboard-concealed-type", text.toUtf8());
     clipboard->setMimeData(mime, QClipboard::Clipboard);
 #else
-    clipboard->setText(text, QClipboard::Clipboard);
+    const QString secretStr = "secret";
+    QByteArray secretBa = secretStr.toUtf8();
+    mime->setText(text);
```

Comment:
Can we move this line and line 53 above the #ifdef, they are the same regardless of OSX or not